Aws/Ec2/Client.py

```diff
@@ -28,3 +29,71 @@ def describe_regions(self) -> Optional[Region]:
         :return: Region object, or None if not found
         """
         return None
+
+    def run_instances(
+            self,
+            instance_type=None,
+            image_id=None,
+            user_data=None,
+            shutdown_behavior=None,
+            network_interfaces=None,
+            min_count=1,
+            max_count=1
+    ):
+        """
+        Start a new EC2 instance
+
+        :param instance_type: Optional Ec2 Instance type/size
+        :type instance_type: Optional[str]
+
+        :param image_id: Optional Instance AMI ID to run the instance with
+        :type image_id: Optional[str]
+
+        :param shutdown_behavior: Optional behaviour of the EC2 instance storage when shutdown
+        :type shutdown_behavior: Optional[str]
+
+        :param user_data: Optional Bash script to run on the instance at startup
+        :type user_data: Optional[str]
+
+        :param network_interfaces: Optional dict for the creation of a network interface to attach to the instance
+        :type network_interfaces: Optional[Dict]
+
+        :param min_count: Optional count for minimum instances to launch
```

Comment:
These are not mandatory

MaxCount (integer) --
[REQUIRED]

The maximum number of instances to launch. If you specify more instances than Amazon EC2 can launch in the target Availability Zone, Amazon EC2 launches the largest possible number of instances above MinCount .

Constraints: Between 1 and the maximum number you're allowed for the specified instance type. For more information about the default limits, and how to request an increase, see How many instances can I run in Amazon EC2 in the Amazon EC2 FAQ.

MinCount (integer) --
[REQUIRED]

The minimum number of instances to launch. If you specify a minimum that is more instances than Amazon EC2 can launch in the target Availability Zone, Amazon EC2 launches no instances.

Constraints: Between 1 and the maximum number you're allowed for the specified instance type. For more information about the default limits, and how to request an increase, see How many instances can I run in Amazon EC2 in the Amazon EC2 General FAQ.